x/data/server/query_data.go

```diff
@@ -107,3 +113,24 @@ func (s serverImpl) ByAttestor(ctx context.Context, request *data.QueryByAttesto
 		Pagination: pageRes,
 	}, nil
 }
+
+func (s serverImpl) getEntry(ctx context.Context, ch *data.ContentHash, iri string) (*data.ContentEntry, error) {
+	dataId, err := s.stateStore.DataIDTable().GetByIri(ctx, iri)
+	if err != nil {
+		return nil, err
+	}
+
+	dataAnchor, err := s.stateStore.DataAnchorTable().Get(ctx, dataId.Id)
+	if err != nil {
+		return nil, err
+	}
+
+	return &data.ContentEntry{
+		Hash: ch,
+		Iri:  iri,
+		Timestamp: &gogotypes.Timestamp{
```

Comment:
this pattern of protobuf ts -> gogo ts seems to happen in quite a few places now, maybe in another PR or this one we can make a util for it?

Reply:
#981